core/templates/dev/head/editor/exploration_history.html

```diff
@@ -20,12 +20,10 @@
   <md-card class="oppia-editor-card">
     <h3>List of Changes</h3>
     <div ng-repeat="versionCheckbox in versionCheckboxArray " class="row">
-      <div class="col-sm-2 col-md-2 col-lg-2">
+      <div class="col-sm-2 col-md-4 col-lg-4">
```

Comment:
@souravbadami Its col-sm-2
Should I make it col-sm-4?
On smaller screen
long username will break in two lines